pkg/client/generated/management/v3/zz_generated_azure_adconfig.go

```diff
@@ -1,50 +1,52 @@
 package client
 
 const (
-	AzureADConfigType                     = "azureADConfig"
-	AzureADConfigFieldAccessMode          = "accessMode"
-	AzureADConfigFieldAllowedPrincipalIDs = "allowedPrincipalIds"
-	AzureADConfigFieldAnnotations         = "annotations"
-	AzureADConfigFieldApplicationID       = "applicationId"
-	AzureADConfigFieldApplicationSecret   = "applicationSecret"
-	AzureADConfigFieldAuthEndpoint        = "authEndpoint"
-	AzureADConfigFieldCreated             = "created"
-	AzureADConfigFieldCreatorID           = "creatorId"
-	AzureADConfigFieldEnabled             = "enabled"
-	AzureADConfigFieldEndpoint            = "endpoint"
-	AzureADConfigFieldGraphEndpoint       = "graphEndpoint"
-	AzureADConfigFieldLabels              = "labels"
-	AzureADConfigFieldName                = "name"
-	AzureADConfigFieldOwnerReferences     = "ownerReferences"
-	AzureADConfigFieldRancherURL          = "rancherUrl"
-	AzureADConfigFieldRemoved             = "removed"
-	AzureADConfigFieldStatus              = "status"
-	AzureADConfigFieldTenantID            = "tenantId"
-	AzureADConfigFieldTokenEndpoint       = "tokenEndpoint"
-	AzureADConfigFieldType                = "type"
-	AzureADConfigFieldUUID                = "uuid"
+	AzureADConfigType                       = "azureADConfig"
+	AzureADConfigFieldAccessMode            = "accessMode"
+	AzureADConfigFieldAllowedPrincipalIDs   = "allowedPrincipalIds"
+	AzureADConfigFieldAnnotations           = "annotations"
+	AzureADConfigFieldApplicationID         = "applicationId"
+	AzureADConfigFieldApplicationSecret     = "applicationSecret"
+	AzureADConfigFieldAuthEndpoint          = "authEndpoint"
+	AzureADConfigFieldCreated               = "created"
+	AzureADConfigFieldCreatorID             = "creatorId"
+	AzureADConfigFieldEnabled               = "enabled"
+	AzureADConfigFieldEndpoint              = "endpoint"
+	AzureADConfigFieldFilterGroupMembership = "filterGroupMembership"
```

Comment:
Again, possible naming convention.  I'd probably slightly prefer AzureADConfigFieldGroupMembershipFilter.

Reply:
Should be fixed automatically when the field name is changed and the code generator re-run.

Reply:
Done with commit f329be6dcd32ad7a6623944341ee72986d771cbd